javascript/javascript2/week1/shortword-daletters.js

```diff
@@ -0,0 +1,25 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function shortWord(array) {
```

Comment:
Function names usually have verbs in the beginning:
`getShortestWord`

Reply:
👍🏻